src/app/teams/teams.service.ts

```diff
@@ -208,11 +208,14 @@ export class TeamsService {
   }
 
   teamNotification(message, user, { team, url }) {
+    const link = url.split(';')[0];
     const userId = user.userId || user._id;
+    const linkParams = message.includes('requested') ? { activeTab: 'applicantTab' } : {};
```

Comment:
Instead of doing a string search on the `message`, we can pass in `type` from `sendNotifications` to this function and use that to test against.